config/helper.go

```diff
@@ -0,0 +1,54 @@
+package config
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+// FlexibleBool is a custom boolean type that can be set in multiple ways
+type FlexibleBool struct {
+	value *bool
+}
+
+// NewFlexibleBool creates a new FlexibleBool with a default value
+func NewFlexibleBool(defaultValue bool) *FlexibleBool {
+	return &FlexibleBool{value: &defaultValue}
+}
```

Comment:
I don't think we should call it a default value since you're setting it explicitly. The default value for a pointer is nil which is "false" for a boolean.
```suggestion
// NewFlexibleBool creates a new FlexibleBool with the given value
func NewFlexibleBool(val bool) *FlexibleBool {
	return &FlexibleBool{value: &val}
}
```